### ISSUE TITLE
USB HDD support

### ISSUE THREAD

**Issue by nickaito** [revvv/Nintendont-XBOX360#12]
Hello friend, I'm very happy to see that your project works well, but I don't have an SD card and the USB doesn't work, unfortunately, could you help me?

**Comment by AngeloCarmelon** [comment]
Yes, I also have the same problem, it would be great if it dealt with that topic

**Comment by NintendoManiac64** [comment]
I actually already reported this over a year ago: issue #4

tl;dr the creator only ever tested using SD cards, not USB drives.

(the one silver lining is that SD cards of triple-digit gigabyte capacities are starting to become stupid cheap relative to the space you'd want for storing GameCube games)

@revvv If you happen to have a microcenter nearby, you can get the following double-combo to have a fully Wii-compatible USB drive setup for only 8 USD (I and the friend mentioned on #4 use the very same Vantec adapter):
- https://www.microcenter.com/site/content/specialofferssd240inf.aspx?web=EMAIL%20OPT%20IN%20-%20DLA_INFLUENCER_MooresLawIsDead_03142022
- https://www.microcenter.com/product/454623/vantec-usb-30-25-sata-hard-drive-adapter-with-case

**Comment by revvv** [comment]
Sorry for my late answer, but github does not always send me an e-mail.

Unfortunately this is not fixable. Duplicate of #4

**Comment by NintendoManiac64** [comment]
I do just want to at least mention that, if you (temporarily) have access to the original disc(s) and a Wii (not Wii U), you can always use Devolution instead which supports wired 360 controllers at the same time as using a USB storage device.

Unfortunately, there never was a version of Devolution released that lacks its copy protection, and I don't think anyone that figured out how to bypass it ever posted their unofficial builds since such things were created back when the dev was still active and years before Nintendont and possibly even the Wii U was even a thing. Furthermore, people nowadays tend to just see Devolution as just being less feature-full and more restrictive than Nintendont and therefore pay little to no attention to it, but this wired 360 controller thing is still one aspect where Devolution is superior despite its other software limitations.
